src/main/resources/static/js/tutorial/tutorial-header.js

```diff
@@ -0,0 +1,206 @@
+  locate_tooltip();
+  show_tooltip();
+});
+// 브라우저 창 크기를 바꿀 시
+// vail, tooltip의 위치 조절
+$(window).resize(() => {
+  locate_vails();
+  locate_tooltip();
+});
+
+// 튜토리얼 - 이전, 다음, 닫기 버튼을 눌렀을 때 적절히 처리
+$("#button_prev").on("click", handle_click_prev);
+$("#button_next").on("click", handle_click_next);
+$("#button_close_tooltip").on("click", hide_vails_and_tooltips);
+
+//////////////////////////////// FUNCTIONS ////////////////////////////////
+
+// 4개 vail 각각의 위치, 크기 설정
+function locate_vails() {
+  let offset = elem_highlight.offset();
+
+  $vailUp.css({
+    top: 0,
+    left: 0,
+    width: "100%",
+    height: `${offset.top}`,
+  });
+  $vailDown.css({
+    top: `${offset.top + elem_highlight.outerHeight(true)}px`,
+    left: 0,
+    width: "100%",
+    height: `calc(100% - ${offset.top + elem_highlight.outerHeight(true)}px)`,
+  });
+  $vailLeft.css({
+    top: `${offset.top}px`,
+    left: 0,
+    width: `${offset.left}px`,
+    height: `${elem_highlight.outerHeight(true)}`,
+  });
+  $vailRight.css({
+    top: `${offset.top}px`,
+    left: `${offset.left + elem_highlight.outerWidth(true)}px`,
+    width: `calc(100% - ${offset.left + elem_highlight.outerWidth(true)}px)`,
+    height: `${elem_highlight.outerHeight(true)}`,
+  });
+}
+
+// vail 모두 보이게 설정
+function show_vails() {
+  for (const vail of vails) vail.css("display", "block");
+}
+
+// tooltip의 위치를 적절히 설정
+function locate_tooltip() {
+  if (!elem_highlight) return;
+  let offset = elem_highlight.offset();
+
+  // 제목, 내용을 n번째 제목, 내용으로 수정
+  $("#tooltip_title").text(array_tooltip_data[elem_index].title);
+  $("#tooltip_content").text(array_tooltip_data[elem_index].content);
+
+  // 위치를 highlight element에 맞게 수정
+  const tooltip_width = 150;
+  let tooltip_left =
+    offset.left - (tooltip_width - elem_highlight.outerWidth(true)) / 2;
+  if (tooltip_left < 0) tooltip_left = offset.left;
+
+  $tooltip.css({
+    position: "absolute",
+    top: `${offset.top + elem_highlight.outerHeight(true) + 10}px`,
+    left: `${tooltip_left}px`,
+    width: `${tooltip_width}px`,
+  });
+}
+
+// tooltip을 화면에 보이게 함
+function show_tooltip() {
+  $tooltip.css({ display: "flex" });
+}
+
+// vail, tooltip을 모두 화면에 보이지 않게 함
+function hide_vails_and_tooltips() {
+  $vailUp.css("display", "none");
+  $vailDown.css("display", "none");
+  $vailLeft.css("display", "none");
+  $vailRight.css("display", "none");
+  $tooltip.css("display", "none");
+}
+
+// tooltip의 '이전' 클릭시 실행되는 함수
+function handle_click_prev() {
+  if (elem_index === 0) return;
+  elem_index -= 1;
+  elem_highlight = array_highlights[elem_index];
+  locate_vails();
+  show_vails();
+  locate_tooltip();
+  show_tooltip();
+}
+
+// tooltip의 '다음' 클릭시 실행되는 함수
+function handle_click_next() {
+  if (elem_index === array_highlights.length - 1) {
+    hide_vails_and_tooltips();
+    return;
+  }
+  elem_index += 1;
+  elem_highlight = array_highlights[elem_index];
+  locate_vails();
+  show_vails();
+  locate_tooltip();
+  show_tooltip();
+}
+});
```

Comment:
let's add newline end of this file.

Reply:
bab4b013a24ca57c981d332f09c1f0014d313359